monkestation/code/modules/blueshift/reagents/deforest.dm

```diff
@@ -213,12 +213,11 @@
 // Reaction to make twitch, makes 10u from 17u input reagents
 /datum/chemical_reaction/twitch
 	results = list(
-		/datum/reagent/drug/twitch = 10,
+		/datum/reagent/drug/twitch = 5,
 	)
 	required_reagents = list(
-		/datum/reagent/impedrezene = 5,
-		/datum/reagent/bluespace = 10,
-		/datum/reagent/consumable/liquidelectricity = 2,
+		/datum/reagent/medicine/adminordrazine = 30,
```

Comment:
this just makes it impossible to make

Reply:
I contemplated on this, but decided not to. Having full projectile immunity is extremely busted, and it's not suppose to be made anyways, which is why it uses adminordrazine.